src/index.css

```diff
@@ -1,13 +1,172 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat+Alternates:wght@200;400;900&display=swap');
+
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  padding: 0;
+  font-family: 'Montserrat Alternates', sans-serif;
+  color: white;
+  background-color: #385892;
+}
+
+nav {
+  padding: 0.5% 6%;
+}
+
+.navLinks {
+  display: flex;
+  flex-direction: row;
+  list-style: none;
+  justify-content: center;
+  align-items: center;
+}
+
+.navLinks img {
+  background-color: black;
+  padding: 6px;
+  border-radius: 50%;
+}
+
+.mainTitle a {
+  color: white;
+  text-decoration: none;
+}
+
+.navIcons {
+  display: flex;
+  flex-direction: row;
+}
+
+.navIcon {
+  margin-left: 50%;
+  cursor: pointer;
+}
+
+.search {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 6%;
+}
+
+.myDate {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  width: 50%;
+}
+
+.date {
+  margin-left: 10%;
+}
+
+.inputSearch {
+  border: none;
+  border-radius: 5px;
+  width: 15%;
+  padding: 0.5% 2%;
+}
+
+.cover {
+  display: flex;
+  flex-direction: row-reverse;
+  height: 18rem;
+  width: 100%;
+  background-image: url('./assets/covid.png');
+  background-position: -900px -400px;
+}
+
+.title {
+  margin-right: 10%;
+  margin-top: 5%;
+}
+
+.logo {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.myCounty {
+  padding: 2%;
+  list-style: none;
+  display: grid;
+  grid-gap: 1rem;
+  grid-template-columns: repeat(2, 1fr);
+  background-color: #4369b2;
+}
+
+@media (min-width: 600px) {
+  .myCounty { grid-template-columns: repeat(4, 1fr); }
+}
+
+@media (min-width: 900px) {
+  .myCounty { grid-template-columns: repeat(6, 1fr); }
+}
+
+.countryCard {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background-color: #324c7c;
+  padding: 5% 10% 0 10%;
+  color: white;
+  border: 0.1px solid rgb(63, 28, 28);
+}
```

Comment:
- [ ] [OPTIONAL] The background color shouldn't be uniform for all the cards, the design given shows otherwise. We can fix this by using something like the code below:
```
div:nth-child(even) {
  background: red;
}
```

Reply:
Thanks, understood 👍